main.tf

```
@@ -114,6 +137,12 @@ resource "aws_iam_policy" "allow_change_password_admin" {
   policy      = "${data.aws_iam_policy_document.allow_change_password.json}"
 }
 
+resource "aws_iam_policy" "allow_manage_access_keys_admin" {
+  name        = "${module.admin_label.id}-permit-manage-keys"
```

```suggestion
  name        = "${module.admin_label.id}-allow-key-management"
```

Suggest using new `label` to follow convention and enforce use of `var.delimiter`